index.js

```diff
@@ -1,18 +1,38 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "green"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const travelerName = document.querySelector('h2')
+travelerName.textContent = traveler.name
 
+const travelerNickName = document.querySelector('em')
+travelerName.textContent = traveler.nickname
 
+const travelerPhoto = document.querySelector('img')
+travelerPhoto.src = traveler.photo
```

Comment:
great! would like the variables to be more specific,

```
travelerNameHeader
travelerNicknameEm
travelerPhotoImg

```
Also don't forget to add the alt attribute for all your images!!

`travPic.alt = traveler.name`